lib/include/up-client-zenoh-cpp/rpc/zenohRpcClient.h

```diff
@@ -26,13 +26,14 @@
 #define _ZENOH_RPC_CLIENT_H_
 
 #include <up-cpp/rpc/RpcClient.h>
-#include <up-client-zenoh-cpp/utils/ThreadPool.h>
-#include <zenoh.h>
+#include <up-cpp/utils/ThreadPool.h>
```

Comment:
I'm not familiar with how this is built today. Seems like via CMake. I would have expected the CMakeLists.txt to contain a reference to the `up-cpp` project. How is this working atm?

Reply:
Answering my own question: it's in [this CMakeLists.txt](https://github.com/eclipse-uprotocol/up-client-zenoh-cpp/blob/6e49a6f6b2a61ebbcb4f6f9c59533a6514ac2681/lib/CMakeLists.txt#L22).